share/qt/Info.plist.in

```diff
@@ -97,9 +97,6 @@
   <key>NSHighResolutionCapable</key>
     <string>True</string>
 
-  <key>LSAppNapIsDisabled</key>
-    <string>True</string>
-  
```

Comment:
Even if it doesn't work in latest versions, why not keep it for older ones?

Reply:
I'm having a hard time finding any Apple documentation about that flag. Do we know where it got deprecated? See also #13362